README.md

```diff
@@ -6,8 +6,6 @@ In the future, the authors plan to turn the script into a Python library that ca
 
 ### Necessary python libraries
 The code was tested with Python 3.7 and higher. The necessary python libraries and their version that were used in development:
-* `argparse`
```

Comment:
These two modules are in standard library, so are always available, no need to mention them here.